README.md

```diff
@@ -116,14 +116,14 @@ Approvers ([@open-telemetry/java-approvers](https://github.com/orgs/open-telemet
 - [Yang Song](https://github.com/songy23), Google
 - [Tyler Benson](https://github.com/tylerbenson), DataDog
 - [Armin Ruech](https://github.com/arminru), Dynatrace
-- [John Watson](https://github.com/jkwatson), NewRelic
 
 *Find more about the approver role in [community repository](https://github.com/open-telemetry/community/blob/master/community-membership.md#approver).*
 
 Maintainers ([@open-telemetry/java-maintainers](https://github.com/orgs/open-telemetry/teams/java-maintainers)):
 
 - [Bogdan Drutu](https://github.com/BogdanDrutu), Google
 - [Carlos Alberto](https://github.com/carlosalberto), LightStep
+- [John Watson](https://github.com/jkwatson), NewRelic
```

Comment:
Minor nit... New Relic is two words. :)

Reply:
@victorNewRelic can you please remove the "request changes"?

Reply:
Also if it is "minor nit" you should not use "request changes" since that blocks the PR to be merged, it is probably not a minor nit in that case :)

Reply:
Luckily you are not approver or maintainer so the blocking PR does not apply here :)

Reply:
This is what happens when you let product managers into github!  😂